worlds/boxes.world.erb

```diff
@@ -1,17 +1,23 @@
 <?xml version="1.0" ?>
+
 <%
   # Boxes benchmark example world
   # SI units (length in meters)
 
+  count = ENV['model_count'].to_i
+  complex = ENV['complex'].to_i
+  dt = ENV['dt'].to_f
+  engine = ENV['engine']
+  collision = ENV['collision'].to_i
```

Comment:
an alternative to environment variables is to set variables on the command-line (as suggested in https://stackoverflow.com/a/39675039 and used in [srcsim](https://github.com/osrf-migration/srcsim-wiki/blob/master/world_generator.md?plain=1#L51)) and you could use `defined?(variable_name)` to see if a value was set or not (see https://stackoverflow.com/a/288726)

Reply:
done, can you please review it again?